docker-image-src/4.3/docker-entrypoint.sh

```diff
@@ -419,6 +419,11 @@ if [ -d /data ]; then
     fi
 fi
 
+if [ -d /licenses ]; then
+    check_mounted_folder_readable "/licenses"
+    : ${NEO4J_dbms_directories_licenses:="/licenses"}
+fi
```

Comment:
What you have here conflicts a bit with what you wrote in the PR message.
So you say we should be able to do:
`-v $HOME/bloom.license:/var/lib/neo4j/licenses/bloom.license`

but here you're expecting the license to be at `/licenses/bloom.license`.

It's better (IMHO) to use `/licenses` as the destination for the license folder because it's shorter and less likely to be typed in wrong, so your code here is fine. If bloom is actually looking for a `$NEO4J_HOME/licenses` folder, or even to make it more compatible either way, I'd suggest putting a symlink under neo4j home to `/licenses`. 
EG `ln -s  /licenses $NEO4J_HOME/licenses`. Same for 4.2.

@eastlondoner do you know how claudio could test the bloom plugin works, or is it enough with the current plugin testing?

Reply:
@jennyowen currently we do not test that any of the plugins "work" because that creates an external dependency on their versions.json and the hosting of the relevant jar files.

This is the same situation as neo4j desktop (as far as I am aware) - we only test that the mechanism works assuming that a valid versions.json and jar exist on the internet.

I also gave Cladio the same feedback on his example and I think he's updated it now (race condition)

Reply:
I fix the PR description using `-v $HOME/bloom.license:/licenses/bloom.license`, actually the bloom plugin will search both in the relative path `${NEO4J_HOME}/licenses/bloom.license` and in the absolute path `licenses/bloom.license`.
I agree to keep the simpler one